### PR TITLE
ingested IntelLanG 2020

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/xml/2020.intellang.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      <author><first>Chamath</first><last>Palihawadana</last></author>
+      <author><first>Kay</first><last>Cooper</last></author>
+      <author><first>Vanessa</first><last>Mendham</last></author>
+      <pages>21–31</pages>
+      <url hash="1c3fdb8f">2020.intellang-1.3</url>
+    </paper>
+    <paper id="4">
+      <title><fixed-case>S</fixed-case>port<fixed-case>S</fixed-case>ett:Basketball - A robust and maintainable data-set for Natural Language Generation</title>
+      <author><first>Craig</first><last>Thomson</last></author>
+      <author><first>Ehud Ehud</first><last>Reiter</last></author>
```
Comment (mjpost) [inline data/xml/2020.intellang.xml:48]: Extra Ehud

File: data/yaml/venues.yaml [hunk 2]
```diff
@@ -416,8 +419,8 @@ jeptalnrecital:
   oldstyle_letter: F
 knlp:
   acronym: knlp
-  name: 'Knowledgeable NLP: Workshop on Integrating Structured Knowledge
-    and Neural Networks for NLP'
+  name: 'Knowledgeable NLP: Workshop on Integrating Structured Knowledge and Neural
+    Networks for NLP'
```
Comment (mjpost) [inline data/yaml/venues.yaml:423]: It seems that every time we run black, it switches this. @xinru1414 can you add this to your list of things to look into?

**Review by mjpost — APPROVED** [review]
Looks good to be merged, once the name issue is fixed.

File: data/yaml/venues.yaml (hunk 2 above)
Comment (xinru1414) [reply data/yaml/venues.yaml:423]: Yes will do

**Pull request merged**